app/src/main/java/nl/tudelft/b_b_w/model/BlockFactory.java

```diff
@@ -31,4 +31,9 @@ public static Block getBlock(String type, String _owner, String _ownHash, String
                 throw new IllegalArgumentException("Invalid type of block: " + type);
         }
     }
+
+    public static Block getBlock(User owner, User user, String ownHash, String previousHashChain, String previousHashSender, String publicKey, boolean revoke) {
```

Comment:
Why did you make another getBlock method? Please explain why you did this. Maybe try adding JavaDoc that explains what this method does.